api-report/datastore.api.md

```diff
@@ -83,10 +85,12 @@ export class FluidDataStoreRuntime extends TypedEventEmitter<IFluidDataStoreRunt
     readonly id: string;
     // (undocumented)
     get IFluidHandleContext(): this;
+    readonly IFluidLoadable?: IFluidLoadable;
```

Comment:
Some number of questions:
- Why there is no entrypoint on IFluidDataStoreChannel? You are doing casting in realizeCore() to get to it - that's the main flow how we expect one to get to this object
- Why it's not a handle directly? Why do we need to wrap it with IFluidLoadable?
- I'd love it to have reasonable name that tells what it is about. Just IFluidLoadable or handle does not tell me anything. Something like `entryPoint` would be good.